picard/ui/ui_options_tags_compatibility_wave.py

```diff
@@ -48,6 +54,7 @@ def setupUi(self, TagsCompatibilityOptionsPage):
     def retranslateUi(self, TagsCompatibilityOptionsPage):
         _translate = QtCore.QCoreApplication.translate
         self.wave_files.setTitle(_("WAVE files"))
+        self.label.setText(_("Picard will tag WAVE files using ID3v2 tags. This is not supported by all software. For compatibility with software that does not support ID3v2 tags in WAVE files, such as Windows Explorer or Windows Media Player, additional RIFF INFO tags can be written to the files. RIFF INFO has only limited support for tags and character encodings."))
```

Comment:
Same here, I would not cite  _Windows Explorer or Windows Media Player_, that's not future-proof, and we don't provide free advertisement for non-free software.